skimage/filters/tests/test_gaussian.py

```diff
@@ -37,13 +37,13 @@ def test_multichannel():
                         [gaussian_rgb_a[..., i].mean() for i in range(3)])
     # Test multichannel = None
     with expected_warnings(['multichannel']):
-        gaussian_rgb_a = gaussian_filter(a, sigma=1, mode='reflect')
+        gaussian_rgb_a = gaussian(a, sigma=1, mode='reflect')
     # Check that the mean value is conserved in each channel
     # (color channels are not mixed together)
     assert np.allclose([a[..., i].mean() for i in range(3)],
                         [gaussian_rgb_a[..., i].mean() for i in range(3)])
     # Iterable sigma
-    gaussian_rgb_a = gaussian_filter(a, sigma=[1, 2], mode='reflect',
+    gaussian_rgb_a = gaussian(a, sigma=[1, 2], mode='reflect',
                                      multichannel=True)
```

Comment:
@ahojnnes Sorry, I am unable to your statement :-(

Can you please elaborate a bit?